buildpipeline/tests/test_pipelines.json

```diff
@@ -22,7 +22,7 @@
                 "HelixJobType": "test/functional/cli/",
                 "TargetsWindows": "true",
                 "Rid": "win-x64",
-                "TargetQueues": "Windows.10.Amd64,Windows.10.Nano.Amd64,Windows.10.Amd64.Core,Windows.7.Amd64,Windows.81.Amd64",
+                "TargetQueues": "Windows.10.Amd64,(Windows.Nano.1803.Amd64.Open)windows.10.amd64.serverrs4.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1803-helix-amd64-05227e1-20190509225944,Windows.10.Amd64.Core,Windows.7.Amd64,Windows.81.Amd64",
```

Comment:
depending on how this list is used (specifically if by powershell) you may find you need to escape the ()s.